SkietbaanBE/SkietbaanBE/Controllers/UserController.cs

```diff
@@ -10,50 +14,63 @@ namespace SkietbaanBE.Controllers
     public class UserController : Controller
     {
         private ModelsContext _context;
-        public UserController(ModelsContext db) {
+        public UserController(ModelsContext db)
+        {
             _context = db;
         }
-
-        //public UserController() { }
-
         // GET: api/User
         [HttpGet]
-        public IEnumerable<User> Get() {
-            var users = _context.Users.ToArray<User>();
-            return users;
+        public IEnumerable<User> Get()
+        {
+           return _context.Users.ToArray<User>();
         }
-
         // GET: api/User/5
         [HttpGet("{id}", Name = "Get")]
-        public User Get(int id) {
-            User user = _context.Users.Find(id);
-            return user;
-        }
-
-        [HttpGet("test/{ret}", Name = "Test")]
-        public string Test(string ret) {
-            return $"Returned : {ret}";
+        public async Task<User> Get(int id)
```

Comment:
Its better to make function names more descriptive to indicate what it is actually doing, like `GetUser`. Do this on all the controller functions in the project

Reply:
noted, changing them now